### PR TITLE
[3.2.0]Update set of contents related to "deployment" and "Installation and start up" topics under FAQ

### PR DESCRIPTION
**Pull request by Chamindu36**
## Purpose
This PR will fix a set of small unnecessary blank spaces and typos related FAQ section of Documentation.
And add will complete Setting up Email Login docs with the latest configurations as suggested in the issue.

### **Removing unnecessary spaces and typos.**

**configuring-secondary-user-stores.md** 
doc4

**key-concepts.md**
doc2

**installation-prerequisites.md**
doc1
doc3

### **Fix for 749**
doc6

## Test environment
Ubuntu 20.04 LTS
Java JDK 1.8_241
APIM 3.2.0
APICTL 3.2.0

### CODE REVIEW

File: en/docs/administer/managing-users-and-roles/managing-user-stores/configuring-secondary-user-stores.md [hunk 1]
```diff
@@ -53,7 +53,7 @@ There are two approaches to configure a secondary user store. It can be configur
 
 4.  Enter a unique domain name with no underscore (\_) characters, and optionally enter a description for this user store.
 
-5.  Enter values for the properties, using the descriptions in the Descriptions column for guidance. The properties that appear vary based on the user store manager class you selected, and there may be additional properties in Optional or Advanced sections at the bottom of the screen. For information on the properties that are used when defining user stores, see [Properties of User Stores](https://docs.wso2.com/display/ADMIN44x/Working+with+Properties+of+User+Stores) .
+5.  Enter values for the properties, using the descriptions in the Descriptions column for guidance. The properties that appear to vary based on the user store manager class you selected, and there may be additional properties in Optional or Advanced sections at the bottom of the screen. For information on the properties that are used when defining user stores, see [Properties of User Stores](https://docs.wso2.com/display/ADMIN44x/Working+with+Properties+of+User+Stores) .
```
Comment (shaniR) [inline en/docs/administer/managing-users-and-roles/managing-user-stores/configuring-secondary-user-stores.md:56]: The properties that appear, vary based on the user store manager class you selected.

File: en/docs/install-and-setup/setup/security/logins-and-passwords/maintaining-logins-and-passwords.md [hunk 2]
```diff
@@ -66,9 +66,32 @@ Use the `<API-M_HOME>/bin/chpasswd.sh` script.
 
     If you are in the super tenant mode the username should be as in the format of `<email>@carbon.super`.
     ``` toml
+    [tenant_mgt]
```
Comment (shaniR) [inline en/docs/install-and-setup/setup/security/logins-and-passwords/maintaining-logins-and-passwords.md:69]: have these been added to the config catalog?

File: en/docs/install-and-setup/setup/security/logins-and-passwords/maintaining-logins-and-passwords.md [hunk 3]
```diff
@@ -66,9 +66,32 @@ Use the `<API-M_HOME>/bin/chpasswd.sh` script.
...
     [apim.throttling]
-    username = "admin@wso2.com@carbon.super"
+    receiver_username = "$ref{super_admin.username}@carbon.super"
+
+    [apim.throttling.jms]
+    username="admin!wso2.com!carbon.super"
+    password = "$ref{super_admin.password}"
     ```
+    
+    !!! note
```
Comment (shaniR) [inline en/docs/install-and-setup/setup/security/logins-and-passwords/maintaining-logins-and-passwords.md:83]: is this note here to describe the properties? If so, please use a table.

**Review by shaniR — CHANGES_REQUESTED** [review]

File: en/docs/install-and-setup/setup/security/logins-and-passwords/maintaining-logins-and-passwords.md (hunk 2 above)
Comment (Chamindu36) [reply en/docs/install-and-setup/setup/security/logins-and-passwords/maintaining-logins-and-passwords.md:69]: [apim.throttling] properties are already added to the config catalog and the [tenant_mgt] will be added with this fix.

File: en/docs/administer/managing-users-and-roles/managing-user-stores/configuring-secondary-user-stores.md [hunk 4]
```diff
@@ -1,87 +1,97 @@
...
+     Do not enter underscore (\_) characters in the domain name, and optionally enter a description for this user store.
 
-5.  Enter values for the properties, using the descriptions in the Descriptions column for guidance. The properties that appear vary based on the user store manager class you selected, and there may be additional properties in Optional or Advanced sections at the bottom of the screen. For information on the properties that are used when defining user stores, see [Properties of User Stores](https://docs.wso2.com/display/ADMIN44x/Working+with+Properties+of+User+Stores) .
+5. Enter values for the properties, using the descriptions in the Descriptions column for guidance. 
 
-    ![]({{base_path}}/assets/img/administer/secondary-user-store-definition.png)
+     The properties vary based on the user store manager class you selected, and there may be additional properties in Optional or Advanced sections at the bottom of the screen. For information on the properties that are used when defining user stores, see [Properties of User Stores](https://docs.wso2.com/display/ADMIN44x/Working+with+Properties+of+User+Stores).
```
Comment (Mariangela) [inline en/docs/administer/managing-users-and-roles/managing-user-stores/configuring-secondary-user-stores.md:63]: This should not point to the Admin guide. Should we move this entire page to the APIM documentation?

File: en/docs/getting-started/key-concepts.md [hunk 5]
```diff
@@ -1,25 +1,22 @@
...
-
-
-
+|  <a name="api-product"></a> API Product                        | A combination of resources from one or more available APIs. An API product allows API product managers to mix and match resources from the available APIs. Similar to using an API, an application that intends to use an API product needs to subscribe to the APIs under a selected business plan. An application that subscribes to an API product gets access to all of its resources, which belongs to one or more of the APIs that are used to create the API product. |
+|  <a name="access-token"></a> Access Token                       | Used in OAuth2.0 bearer token-based authentication to allow a consumer to access an API. It is a string that is passed as an HTTP header of an API request. WSO2 API Manager supports JWT formatted self-contained access tokens and regular opaque access tokens. |
```
Comment (Mariangela) [inline en/docs/getting-started/key-concepts.md:11]: @ChamodDamitha , based on https://github.com/wso2/docs-apim/pull/1962 should we remove the following from this sentence?
"and regular opaque access tokens."

File: en/docs/reference/config-catalog.md [hunk 6]
```diff
@@ -4389,3 +4389,59 @@ persistent_notifier.password = "root"</code></pre>
...
+
+<div class="mb-config-catalog">
+    <section>
+        <div class="mb-config-options">
+            <div class="superfences-tabs">
+
+                <input name="16" type="checkbox" id="_tab_16">
+                <label class="tab-selector" for="_tab_16"><i class="icon fa fa-code"></i></label>
+                <div class="superfences-content">
+                    <div class="mb-config-example">
```
Comment (Mariangela) [inline en/docs/reference/config-catalog.md:4405]: @Chamindu36 , Nothing appears when the "view sample" button is clicked.

Screenshot 2020-10-14 at 15 54 00

Comment (Chamindu36) [reply en/docs/reference/config-catalog.md:4404]: Resolved

**Review by Mariangela — APPROVED** [review]

File: en/docs/getting-started/key-concepts.md (hunk 5 above)
Comment (ChamodDamitha) [reply en/docs/getting-started/key-concepts.md:11]: yes, we should remove

**Pull request merged**